recipes/oso/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "oso" %}
+{% set build_number = 0 %}
+{% set version = "0.26.1" %}
+{% set min_rust = "rust >=1.46" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/osohq/oso/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 412b62ca46b649cd2b0ddfd9027180eeda9aa8d4f059da44978fa56ac103e90a
+
+build:
+  number: {{ build_number }}
+  script:
+    - bash {{ RECIPE_DIR }}/build_oso.sh
+
+requirements:
+  build:
+    - {{ compiler('c') }}                    # [unix]
+    - {{ compiler('cxx') }}                  # [unix]
+    - {{ compiler('m2w64_c') }}              # [win]
```

Comment:
I swapped to `msvc_c` instead of `m2w64_c`, etc and I got this: `conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("msvc_cxx_win-64"), MatchSpec("msvc_c_win-64")}`

Not sure if that's the wrong string to use there or what though but I've swapped back for now

Reply:
just `c`.

Reply:
example: https://github.com/conda-forge/python-wasmer-split-feedstock/blob/main/recipe/meta.yaml